beetsplug/fetchart.py

```diff
@@ -41,6 +41,10 @@
 CONTENT_TYPES = ('image/jpeg', 'image/png')
 DOWNLOAD_EXTENSION = '.jpg'
 
+CANDIDATE_BAD = 0
+CANDIDATE_EXACT = 1
+CANDIDATE_DOWNSCALE = 2
```

Comment:
enums sound a little overkill for something that I presume will only be used internally by the plugin.